source/sync/data-model/update-data-model/update-schema-production-apps.txt

```diff
@@ -1,20 +1,430 @@
+     - Yes
+     - Yes
+   * - :ref:`Change whether property is optional or required <schema-update-change-property-status>`
+     - Schema Versioning (Supported)
+     - Yes
+     - Yes
+   * - :ref:`Rename an object type <schema-update-change-object-type-name>`
+     - Breaking (Unsupported)
+     - Yes
+     - No
+   * - :ref:`Rename a property <schema-update-change-property-name>`
+     - Breaking (Unsupported)
+     - Yes
+     - No
+   * - :ref:`Change property type <schema-update-change-property-type-but-keep-name>`
+     - Breaking (Unsupported)
+     - Yes
+     - No
+
+.. tip:: Remap a Name Instead of Renaming
+
+   Several Device SDKs support remapping a property or class name. Because
+   renaming is considered a breaking change, we recommend remapping
+   through the client code when possible. To learn how, refer to the following
+   .. TODO: add link
+
+.. _update-data-model-production-apps-prerequisites:
+
+Prerequisites
+-------------
+
+To use schema versioning in App Services, your App Services App must meet the
+following requirements:
+
+- Device Sync must be enabled. To learn how, refer to :ref:`enable-sync`.
+- Development Mode must be disabled. To learn how, refer to :ref:`Disable
+  Development Mode <enable-development-mode>`.
+
+To set schema versioning from the client, your app must also use a minimum
+Device SDK version:
+
+.. TODO: confirm sdk versions & link out to SDK docs re. setting schema version
+
+- C++ SDK - unsupported
+- Flutter SDK v2.1.0
+- Java SDK v10.18.0
+- Kotlin SDK -
+- .NET SDK v11.7.0
+- Node.js SDK v12.6.0
+- React Native SDK v12.6.0
+- Swift SDK v10.46.0
+
+.. _update-data-model-production-app-procedure:
+
+Make Changes with Schema Versioning
+-----------------------------------
+
+The following describes how to make changes to your server-side schema through
+the App Services UI when schema versioning is enabled. For information on the
+specific changes you can make to your schema, refer to
+:ref:`sync-schema-overview`.
+
+.. procedure::
+
+   .. step:: Make Schema Changes in App Services UI
+
+      In the App Services UI, navigate to the :guilabel:`Schema` page.
+
+      1. From the :guilabel:`Collections` tab, select the schema that you want
+         to modify.
+
+      .. note:: Define Default Values When Possible
+
+         If your changes impact the default values of a property (for example,
+         when adding a new required property), we recommend that you define a
+         default value. To learn how, refer to
+         :ref:`schema-update-add-a-default-value`.
+
+      2. Do one of the following, depending on whether you are using deployment
+         drafts in your app:
+
+         - If drafts are disabled, click :guilabel:`Save`.
+         - If drafts are enabled, click :guilabel:`Save Draft`. From the
+           banner, click :guilabel:`Review Draft & Deploy`.
+
+      4. If you *only* made non-breaking changes, your changes are automatically
```

Comment:
If you adopt the alternate language I suggest above (or different language) we'd need to change it here to "Fully-Compatible" or whatever else we use.